dxr/plugins/xbl/analyzer.py

```diff
@@ -0,0 +1,78 @@
+from warnings import warn
+import xml.parsers.expat as expat
```

Comment:
Any particular reason you chose expat? I've never seen it used before. OTOH, it doesn't make it hard for us to move to PyPy, unlike lxml. I'm a little nervous that it hasn't been updated since 2012 and it written in C and so probably has security holes, but presumably Python takes some measure of responsibility for it since it's in the stdlib.